hack/docker-builder/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM fedora:28
+
+ENV LIBVIRT_VERSION 4.2.0
+
+COPY fedora-virt-preview.repo /etc/yum.repos.d/fedora-virt-preview.repo
+
+RUN dnf -y install libvirt-devel-${LIBVIRT_VERSION} make git mercurial sudo gcc findutils gradle rsync-daemon rsync qemu-img protobuf-compiler && \
```

Comment:
we don't need many of these dependencies for ovs-cni. Please iterate and remove what is not needed.

Reply:
Same, please hint.